tests/flow/test_function_calls.py

```
@@ -2119,146 +2119,146 @@ def test83_Replace(self):
-    #    try:
-    #        actual_result = self.graph.query(query)
-    #    except redis.ResponseError as e:
-    #        self.env.assertContains("Division by zero", str(e))
+    def test84_RandomUUID(self):
+        query = "RETURN randomUUID()"
+        actual_result = self.graph.query(query).result_set[0][0]
+        self.env.assertEquals(actual_result[8], '-')
+        self.env.assertEquals(actual_result[13], '-')
+        self.env.assertEquals(actual_result[14], '4')
+        self.env.assertEquals(actual_result[18], '-')
+        if not actual_result[19] in ['8', '9', 'a', 'b']:
```

Ruff (0.14.10)

Test for membership should be `not in`

Convert to `not in`

(E713)